### PR TITLE
update domain index

### PR DESCRIPTION
**Pull request by czue**

### CODE REVIEW

File: corehq/pillows/mappings/domain_mapping.py [hunk 1]
```diff
@@ -1,4 +1,4 @@
-DOMAIN_INDEX = "hqdomains_391fc13ad99b11e48da43c970e7be43b"
+DOMAIN_INDEX = "hqdomains_20150403_1448"
```
Comment (czue) [inline corehq/pillows/mappings/domain_mapping.py]: @dimagi/team-commcare-hq this is a new convention i'm proposing to make indices more readable: [index]_[YYMMDD_HHMM]

there's no reason I can think of why they need to be guids.

Comment (dmyung) [inline corehq/pillows/mappings/domain_mapping.py]: Does this affect the calculation of a diff though? or is the delta calculation done on the fly with what's in source control (live) and what changes it calculates from the Case object?

Comment (czue) [inline corehq/pillows/mappings/domain_mapping.py]: we've been phasing out the dynamic hash generation off the mapping, if that's what you're asking

Comment (dmyung) [inline corehq/pillows/mappings/domain_mapping.py]: ah cool, I never liked that that :see_no_evil:

**Comment by esoergel** [comment]
:+1:

File: corehq/pillows/mappings/domain_mapping.py (hunk 1 above)
Comment (millerdev) [inline corehq/pillows/mappings/domain_mapping.py]: +1 for readability

**Pull request merged**